### PR TITLE
libz blitz: turn references to other semver items into links

### PR DESCRIPTION
**Pull request by muzikmoe**
Rebase pull request #143 on `master`

### CODE REVIEW

**Comment by udoprog** [comment]
Hey!

I was looking through the documentation on `lib.rs`, and I would really like to turn all references to the semver spec (or `the spec`) into links as well. Sections 7 and 8 are mentioned.

I also note that you are using rustfmt which introduces quite a bit of changes outside of documentation. I don't think it's unusual to hook rustfmt into your editor, so I've opened #147 to try to address this.

File: src/version_req.rs [hunk 1]
```diff
@@ -7,6 +7,7 @@
 // <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
+//
```
Comment (steveklabnik) [inline src/version_req.rs]: extra `//` snuck in here, would you mind removing this please?

File: src/version_req.rs [hunk 2]
```diff
@@ -84,13 +90,13 @@ enum WildcardVersion {
...
 #[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
 enum Op {
-    Ex, // Exact
-    Gt, // Greater than
-    GtEq, // Greater than or equal to
-    Lt, // Less than
-    LtEq, // Less than or equal to
-    Tilde, // e.g. ~1.0.0
-    Compatible, // compatible by definition of semver, indicated by ^
+    Ex,                        // Exact
```
Comment (steveklabnik) [inline src/version_req.rs]: I thought doing this was non-standard style; I generally prefer not aligning them. I haven't reviewed the relevant Libz Blitz guidelines though

Comment (KodrAus) [reply src/version_req.rs]: I don't think we really cover formatting in the guidelines, but personally also prefer them not aligned.

**Comment by opilar** [comment]
@muzikmoe it's in conflict again. Do you mind if I rebase your change without reformatting?

**Comment by muzikmoe** [comment]
@opilar that's ok, do whatever you want to do.

**Comment by steveklabnik** [comment]
I've merged in the rebased version. Thanks so much!

**Pull request closed without merging**